components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/RecoveryApiV2.java

```diff
@@ -193,4 +247,109 @@ private RecoveryResponse recover(RecoveryRequest recoveryRequest, String tenantD
                 localVarHeaderParams, localVarFormParams, localVarAccept, localVarContentType, localVarAuthNames,
                 localVarReturnType);
     }
+
+    private ResendResponse resend(ResendRequest resendRequest, String tenantDomain, Map<String, String> headers,
+                                  String localVarPath) throws ApiException {
+
+        // Verify if the required parameter 'recoveryRequest' is set.
+        if (resendRequest == null) {
+            throw new ApiException(400, "Missing the required parameter 'resendRequest' when requesting recovery");
+        }
+        if (StringUtils.isBlank(tenantDomain)) {
+            tenantDomain = MultitenantConstants.SUPER_TENANT_DOMAIN_NAME;
+        }
+        basePath = IdentityManagementEndpointUtil.getBasePath(tenantDomain,
+                IdentityManagementEndpointConstants.UserInfoRecovery.RECOVERY_API_V2_RELATIVE_PATH);
+        apiClient.setBasePath(basePath);
+        List<Pair> localVarQueryParams = new ArrayList<Pair>();
+        Map<String, String> localVarHeaderParams = new HashMap<String, String>();
+        if (MapUtils.isNotEmpty(headers)) {
+            localVarHeaderParams.putAll(headers);
+        }
+        Map<String, Object> localVarFormParams = new HashMap<String, Object>();
+        final String[] localVarAccepts = {
```

Comment:
do we need to define a variable here?

Reply:
Removed all redundent variables with https://github.com/wso2/carbon-identity-framework/pull/5601/commits/3dcd28533c228416a037539e869ab90c340ef341